src/main/java/frc/robot/Orchestrator.java

```diff
@@ -0,0 +1,286 @@
+            Commands.parallel(
+                    indexer.setPercent(IndexerConstants.INDEX_SPEED.get()), intake.intake())
+                .until(indexer::getLimitSwitchPressed)
+                .withTimeout(10))
+        .finallyDo(() -> indexer.setPercent(-0.6).withTimeout(2));
+  }
+
+  /**
+   * Turns on the intake while driving for 2 seconds, both happening in parallel.
+   *
+   * @return The command to move the robot and intake.
+   */
+  public Command driveWhileIntaking() {
+    return Commands.parallel(
+        intake.intake().until(indexer::getLimitSwitchPressed),
+        Commands.run(
+                () -> drive.runVelocity(new ChassisSpeeds(Units.inchesToMeters(10), 0.0, 0.0)),
+                drive)
+            .withTimeout(2));
+  }
+
+  /**
+   * Turn on the intake when we see a note.
+   *
+   * @return The command to intake after a note is seen.
+   */
+  // Intakes after seeing note with Pixy2.
+  public Command basicVisionIntake() {
+    return Commands.sequence(
+        indexer.setVolts(4),
+        arm.toSetpoint(ArmConstants.OFFSET),
+        Commands.waitUntil(() -> arm.atSetpoint() && pixy2.seesNote()).withTimeout(2),
+        new ProxyCommand(
+            () ->
+                new StraightDriveToPose(
+                    new Pose2d(
+                        drive.getPose().getTranslation(),
+                        drive
+                            .getRotation()
+                            .plus(
+                                AllianceFlipUtil.apply(Rotation2d.fromDegrees(pixy2.getAngle())))),
+                    drive)),
+        intake.intake().until(indexer::getLimitSwitchPressed));
+  }
+
+  /* TODO: Complete once pixy is done. Will drive towards note using the angle and distance supplied by the pixy2.
+  Will use intakeBasic in parallel. */
+  public Command fullVisionIntake() {
+    return null;
+  }
+
+  /**
+   * Expels the intake if we want to get rid of a note.
+   *
+   * @return The command to release a note in the intake.
+   */
+  public Command expelIntake() {
+    return intake.release();
+  }
+
+  /**
+   * Expels the intake if we want to get rid of a note in shooter and indexer.
+   *
+   * @return The command to release a note in the shooter and indexer.
+   */
+  public Command expelShindex() {
+    return Commands.parallel(shooter.manualShoot(-0.2 * 12), indexer.setPercent(-1.0));
+  }
+
+  /**
+   * Expels a note from the whole robot.
+   *
+   * @return The command to release a note from all subsystems.
+   */
+  public Command expelFullRobot() {
+    return Commands.parallel(expelIntake(), expelShindex());
+  }
+
+  /**
+   * Uses intakeBasic and shootBasic in order to shoot a note while lined up with the speaker,
+   * doesn't move the arm.
+   *
+   * @return The command to intake a note and then shoot that note, more for testing purposes.
+   */
+  public Command intakeAndShootSpeaker() {
+    return Commands.sequence(intakeBasic(), shootBasic());
```

Comment:
We are almost never intaking from the shoot basic position, so this command is clutter.